Morphic/Morphic.pier

```diff
@@ -323,21 +330,21 @@ all crossMorphs answer ==true== to the ==handlesMouseDown:== message.
 
 @@todo Add this method to ==CrossMorph==:
 
-[[[caption=Declaring that ==CrossMorph== will react to mouse clicks.
-CrossMorph>>>handlesMouseDown: anEvent
-	^true
+[[[language=smalltalk|caption=Declaring that ==CrossMorph== will react to mouse clicks|label=scr:mouseEvent
+CrossMorph>>#handlesMouseDown: anEvent
```

Comment:
I think I saw that notation. But I never saw `>>>`. Two yes but not three. I'm not sure about the `#`, if we consider the name of the method as a symbol or not. I'll ask to Stephane what to use.